### DISCUSSION TITLE
pothos prisma object type

### DISCUSSION THREAD

**Discussion by AnirudhKaranth** [prisma/prisma#22841] [Q&A]
### Question

I have 3 models... model1, model2 and model3. I need to write a query to return a response by combining the data of all 3 models.
I defined a custom objectType using class. But I get error when I assign it to type field Type 
'ObjectRef<Quizsubs, Quizsubs>' is not assignable to type 'PrismaRef<PrismaModelTypes> | [PrismaRef<PrismaModelTypes>] | keyof PrismaTypes | [keyof PrismaTypes]'.
  Type 'ObjectRef<Quizsubs, Quizsubs>' is missing the following properties from type 'PrismaObjectRef<PrismaModelTypes, {}>': modelName, addBrand, hasBrand, [prismaModelKey], [abstractReturnShapeKey]

My Question is how to convert objectType to prsimaObject.
```
t.prismaField({
        type: customObj,
        args: {
            id: t.arg({
                type: "String",
                required: true
            })
        },
       resolve: async (query, root, args, ctx, info) => {
            const ans1 = await ctx.prisma.model1.findMany({
                  where:{    
                        Question:{
                            id:args.id
                        }
                    }
                  
                   
                })

const ans2 = await ctx.prisma.model2.findMany({
                  where:{    
                        Question:{
                            id:args.id
                        }
                    }
                  
                   
                })
  }
const ans3 = await ctx.prisma.model3.findMany({
                  where:{    
                        Question:{
                            id:args.id
                        }
                    }
                  
                   
                })

// code to combine the results and return it

})
  ```

**Comment by ludralph** [comment]
Hi @AnirudhKaranth 

This question looks more like a `Pothos-Prisma` specific question and not a Prisma ORM question.

Can you please ask the question in [Pothos GraphQL](https://github.com/hayes/pothos/discussions) discussion forum?
